OxfordBodleian/Juel-Jensen/BDLaethf24.xml

```diff
@@ -33,21 +36,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <idno>Juel-Jensen 6</idno>
                         </altIdentifier>
                     </msIdentifier>
-                    
+
                     <msContents>
                         <summary/>
-                        
+
                         <!--<msItem xml:id="ms_i1">
-                      
+
                         </msItem>-->
-                        
+
                     </msContents>
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>A large hole, presumably resulting from the parchment making process, is found on <locus target="#2"/>.</p>
```

Comment:
Maybe the word "tear" would be better than "hole", because now I first thought of an intentional hole. Or maybe now :)

Reply:
Since it has a rounded shape, I would say it is an hole, but I am open. What do others think?

Reply:
To me, "hole" is the usual term for this phenomenon (I also have the association with roundedness). They are differenciated from holes or tears that happened at a later stage in the life of the manuscript by their place in the description.

Reply:
My point was that it would be good to distinguish these (accidental) "holes" from the (intentional) "holes" that, for example, resulted from the binding. It would be possible to use another term for the other piercings as well… or maybe just to specify using "accidental"/"intentional"... or maybe there is no need to clarify, and the average reader will understand what is meant (although it took me some seconds of thought to understand) :)

Reply:
Doesn't the reference to the circumstances in which the holes originated (parchment making) help to clear their nature?

Reply:
My feeling is that the process of parchment making could also leave intentional holes in the parchment, depending on which cuts are used in the manuscript :)  But I think you can use the formulation that you had – after all, Dorothea and Massimo found it clear, so it is good :)